ext/hash/tests/sensitive_parameter.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Test that sensitive parameters are marked sensitive.
+--FILE--
+<?php
+try {
+    var_dump(hash_equals('foo', null));
+} catch (\Throwable $e) {
+    echo $e, PHP_EOL;
```

Comment:
This specifically tests two functions:

- hash_equals because it has two parameters.
- hash_hmac because the sensitive parameter is not the first parameter

Without the try-catch the test would exit after the first exception and thus would not test the second function. It does not appear useful to me to split this into two entirely separate `.phpt` files.

Reply:
Yes, you are right!